### ISSUE TITLE
plugin: Outside of render distance esp/tracer/waypoint pointing at a player (for following a group/friends without losing track of them)

### ISSUE THREAD

**Issue by FU3X** [lambda-client/lambda#552]
**Is your feature request related to a problem? Please elaborate.**

On 2b2t i constantly had a problem where my friend that im following, would go outside of render distance and we would always have to ask for eachothers coordinates, and i know many other players who travel together face the same issues.

**Describe the solution or feature you'd like implemented.**

Using internet communication to securely establish a direct connection to your friend(s), this can be done by port forwarding or a p2p network. 

like tor; if you guys allowed links to establish the connection, permanent onion urls can be generated and point to whatever port this plugin will use, and the ip or link will be placed in the first box. Then a authentication key will be in the second box, there can be multiple of these keys for a single endpoint, like for example multiple users for connecting to a different session. 

make the user be able to add more connections and toggle them, if the connection has the same exact credentials for each person who configured it, it will be like a group.

basically a person can host this and and he has to open a session on his minecraft client, he will need to give his friends a ip or link and a room key for them to connect

Finally the main point: after player(s) have successfully been able to connect to eachother, either a tracer, dot, nametag, waypoint or anything can be rendered at the player's exact position no matter the distance.

maybe this featured can be called shared esp, player tracker or something related 

**Describe alternatives you've considered.**

I couldn't find anything about this 

**Additional context:**
n/a

**Comment by FU3X** [comment]
> Just send the coordinates and wait

no that's not even the point, also i just said it's titus to jump through so many hoops like that
